kube/src/config/file_config.rs

```diff
@@ -52,12 +52,17 @@ pub struct NamedCluster {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Cluster {
     pub server: String,
+    #[serde(rename = "tls-server-name")]
+    pub tls_server_name: Option<String>,
     #[serde(rename = "insecure-skip-tls-verify")]
     pub insecure_skip_tls_verify: Option<bool>,
     #[serde(rename = "certificate-authority")]
     pub certificate_authority: Option<String>,
     #[serde(rename = "certificate-authority-data")]
     pub certificate_authority_data: Option<String>,
+    #[serde(rename = "proxy-url")]
+    pub proxy_url: Option<String>,
```

Comment:
oh, is this new? i didn't see any good info from a quick google. are we meant to actually proxy if this is set?

Reply:
Technically we don't support proxying at the moment. We could include this in the config, but we should have a comment caveating that it's not used in our client yet.

Reply:
Would it be better to omit these for now?  My goal here is to just avoid the panics with the minikube generated config : )

Reply:
Yeah, probably best to avoid it to minimise confusion until we have support for it :-)

Reply:
👍